src/redis-cli.c

```diff
@@ -9418,9 +9606,10 @@ static long getLongInfoField(char *info, char *field) {
 }
 
 /* Convert number of bytes into a human readable string of the form:
- * 100B, 2G, 100M, 4K, and so forth. */
-void bytesToHuman(char *s, size_t size, long long n) {
+ * 1003B, 4.03K, 100.00M, 2.32G, 3.01T */
+char* bytesToHuman(char *s, size_t size, long long n) {
```

Comment:
```suggestion
char *bytesToHuman(char *s, size_t size, long long n) {
```

Reply:
on the other hand, i'm a fan of this implement that returns the original s.
although it makes cleanPrintfln save two lines, it still seems werid.

Reply:
I used the return value of `bytesToHuman()` 7 times when calling `cleanPrintfln()`. I thought it will be convenient to return a `char*`. We have the choice to use the return value or not.
Let me know if you prefer to have:
```
    bytesToHuman(buf, sizeof(buf);
    line_count += cleanPrintfln("Keys size:    %s", buf, total_size));
```
instead of:
```
    line_count += cleanPrintfln("Keys size:    %s", bytesToHuman(buf, sizeof(buf), total_size));
```
I am ok with both.

Reply:
in that case it's better update the comment as well.

Reply:
Good point, I will add in the comment that the return value is the original *s pointer.